h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -304,6 +341,10 @@ private static final int calculateLocalConnectionCount(final int maxTotalConnect
       //Make sure at least some connections are available to a node
       return Math.max(conPerNode, MIN_CONNECTIONS_PER_NODE);
     }
+
+    static int getOptimalConnectionPerNode(final int cloudSize, final short nThreads) {
```

Comment:
Javadoc + maybe we can keep private modifier in place.  Or is this because of tests ?

Reply:
done